pymongo/network.py

```diff
@@ -162,15 +164,34 @@ def command(
 
     if max_bson_size is not None and size > max_bson_size + message._COMMAND_OVERHEAD:
         message._raise_document_too_large(name, size, max_bson_size + message._COMMAND_OVERHEAD)
-
+    encoding_duration = datetime.datetime.now() - start
+    command_logger = logging.getLogger("pymongo.command")
+    # TODO: add serverConnectionId
```

Comment:
Is this a TODO for this PR or a TODO to be deferred for another? If it's for another, we can make a JIRA ticket to track and place it here.

Reply:
Whoops missed these comments: I'll be opening https://jira.mongodb.org/browse/PYTHON-2822 shortly to resolve this.

Reply:
Opened: https://github.com/mongodb/mongo-python-driver/pull/1438